drivers/scale_host.go

```diff
@@ -44,8 +44,21 @@ func (s *ScaleHostDriver) ValidatePayload(conf interface{}, apiClient *client.Ra
 		return http.StatusBadRequest, fmt.Errorf("Invalid amount: %v", config.Amount)
 	}
 
-	if config.HostSelector == nil {
-		return http.StatusBadRequest, fmt.Errorf("HostSelector not provided")
+	if config.HostSelector == nil && config.HostTemplateID == "" {
+		return http.StatusBadRequest, fmt.Errorf("HostSelector or HostTemplateID is not provided")
+	}
+
+	if config.HostSelector != nil && config.HostTemplateID != "" {
+		return http.StatusBadRequest, fmt.Errorf("HostSelector and HostTemplateID cannot both exist in one config")
+	}
+
+	hostTemplate, err := apiClient.HostTemplate.ById(config.HostTemplateID)
```

Comment:
This message will be returned to the user, so it needs to be more readable.
```
"Error getting hostTemplate resource"
```
Also, you should log the error here via logrus.Errorf()